spec/datadog/profiling/collectors/cpu_and_wall_time_worker_spec.rb

```diff
@@ -56,13 +60,16 @@
   end
 
   describe '#start' do
+    let(:expected_worker_initialization_error) { nil }
+
     subject(:start) do
       cpu_and_wall_time_worker.start
       wait_until_running
     end
 
+    @skip_cleanup = false
```

Comment:
I'm not sure this is in the right scope, actually, since it's outside everything in the dsl. Might not be doing anything?